submissions/serializers.py

```diff
@@ -90,6 +90,21 @@ class ScoreSerializer(serializers.ModelSerializer):
     # Ensure that the created_at datetime is not converted to a string.
     created_at = DateTimeField(format=None, required=False)
 
+    annotations = serializers.SerializerMethodField()
+    def get_annotations(self, obj):
+        """
+        Inspect ScoreAnnotations to attach the annotation_type of any found annotations.
+        """
+        annotations = ScoreAnnotation.objects.filter(score_id=obj.id)
+        return [
```

Comment:
Isn't there a default serializer for dicts? I would expect it to serialize whatever fields are present (so this code does not need to change in the number of fields in the dict changes in the future).